src/Disks/IDisk.h

```diff
@@ -564,6 +549,27 @@ inline String directoryPath(const String & path)
     return fs::path(path).parent_path() / "";
 }
 
+static std::string_view normalizePath(std::string_view path)
+{
+    if (!path.empty() && path.ends_with('/'))
+        return path.substr(0, path.size() - 1);
+    return path;
+}
+
+inline bool isPrefixPath(std::string_view prefix, std::string_view path)
+{
+    prefix = normalizePath(prefix);
+    path = normalizePath(path);
+
+    return path.starts_with(prefix) &&
+        (path.size() == prefix.size() || path[prefix.size()] == '/');
+}
+
+inline bool arePathsEqual(std::string_view lhs, std::string_view rhs)
+{
+    return normalizePath(lhs) == normalizePath(rhs);
+}
```

Comment:
Probably this will not be needed

```suggestion
```

Reply:
Actually it's the only difference, since other changes from outdated version, so let's not to fix it here